spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindRefComparisonTest.java

```diff
@@ -0,0 +1,97 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.DetectorFactory;
+import edu.umd.cs.findbugs.DetectorFactoryCollection;
+import edu.umd.cs.findbugs.SystemProperties;
+import edu.umd.cs.findbugs.annotations.Confidence;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+public class FindRefComparisonTest extends AbstractIntegrationTest {
+
+    @AfterAll
+    static void tearDown() {
+        SystemProperties.removeProperty("findbugs.refcomp.reportAll");
+        setPriorityAdjustment(0); // reset priority
+    }
+
+    @Test
+    void testComparisonsNotEnabled() {
+        /**
+        * When there is no priority adjustment, and we enable ref comparison for
+        * all, we only get an integer ref comparison bug.
+        */
+        setPriorityAdjustment(0); // reset priority
+        SystemProperties.setProperty("findbugs.refcomp.reportAll", "true"); // enable ref comparison
+
+        performAnalysis("RefComparisons.class");
```

Comment:
Can you please add the nested classes here and every other `performAnalysis()` call in this class as well: `"RefComparisons$MyClass1.class"`, `"RefComparisons$MyClass2.class"`, `"RefComparisons$MyClass3.class"`?

Reply:
added as part of [df5db48](https://github.com/spotbugs/spotbugs/pull/2988/commits/df5db48bf8190dfff44297d9b823c63b2606b199)